app/src/main/java/org/oppia/app/player/state/itemviewmodel/ReturnToTopicButtonViewModel.kt

```diff
@@ -0,0 +1,10 @@
+package org.oppia.app.player.state.itemviewmodel
+
+import org.oppia.app.player.state.listener.PreviousNavigationButtonListener
+import org.oppia.app.player.state.listener.ReturnToTopicNavigationButtonListener
+
+/** [StateItemViewModel] for both previous state navigation and navigating back to the topic containing this lesson. */
+class ReturnToTopicButtonViewModel(
+  val hasPreviousButton: Boolean, val previousNavigationButtonListener: PreviousNavigationButtonListener,
+  val returnToTopicNavigationButtonListener: ReturnToTopicNavigationButtonListener
```

Comment:
This should come in multiple lines like this:
```
  val hasPreviousButton: Boolean, 
  val previousNavigationButtonListener: PreviousNavigationButtonListener,
  val returnToTopicNavigationButtonListener: ReturnToTopicNavigationButtonListener
```

Reply:
+1

Reply:
Done.